pkgs/by-name/ko/kontroll/package.nix

```diff
@@ -1,25 +1,47 @@
 {
   lib,
+  stdenv,
   rustPlatform,
   fetchFromGitHub,
   protobuf,
+  darwin,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "kontroll";
-  version = "1.0.1";
+  version = "1.0.3";
 
   src = fetchFromGitHub {
     owner = "zsa";
     repo = "kontroll";
     rev = version;
-    hash = "sha256-k7Twbjl8umk3PeIv3ivCLdhZFgTTV8WdfIAoGAD/pEk=";
+    hash = "sha256-SSKAueJbDhziEwM6VcMKgvgdNdZWCXVJUj0P4EQtiKs=";
   };
 
-  cargoHash = "sha256-931PmSvcXTjxdIQUCSxLwFnaHgRrFzA/v6yYTvkjNoQ=";
+  cargoHash = "sha256-D0lpbJdA8IcDebU7KWG/Kw/rjuJZ6ZVxxhlDHVS1Hfg=";
 
   nativeBuildInputs = [ protobuf ];
 
+  # Integration tests ($src/tests) are not complete (trait impls are `todo!()`)
+  # so we just run the rest
+  cargoTestFlags = [
+    "--lib"
+    "--bins"
+    "--examples"
+  ];
+
+  # Needed for compiling the examples
+  checkInputs = lib.optionals stdenv.isDarwin (
+    with darwin.apple_sdk.frameworks;
+    [
+      Foundation
+      AppKit
+      Vision
```

Comment:
ofborg doesn't find this library. Are you sure it is supposed to be there?

Reply:
Just tried, sharing output. Check the last 10 lines or so for the relevant failure info.

The one that fails is one of the examples, but this should be available as a Darwin framework anyways no?

[nix-build-kontroll.log](https://github.com/user-attachments/files/17580224/nix-build-kontroll.log)

Reply:
I see. Then I'm wondering why ofborg has trouble:
```
       error: undefined variable 'Vision'

       at /ofborg/checkout/3/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-0/pkgs/by-name/ko/kontroll/package.nix:39:7:

           38|       AppKit
           39|       Vision
             |       ^
           40|       AVFoundation
```